src/actions/account.js

```diff
@@ -164,6 +164,20 @@ async function getAccounts(tokens, options) {
   }, Promise.resolve({}));
 }
 
+export const updateEnabledTokensAccounts = token => async (dispatch, getState) => {
```

Comment:
Now the name (plural) doesn't match the implementation (singular)
```suggestion
export const updateEnabledTokenAccount = token => async (dispatch, getState) => {
```